src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/EducationInNumbersServiceTests.cs

```diff
@@ -1079,43 +1086,50 @@ public async Task Reorder_Success()
     [Fact]
     public async Task Reorder_WithAmendment_Success()
     {
-        var page1V0 = new EducationInNumbersPage { Id = Guid.NewGuid(), Slug = "page-1", Version = 0, Order = 0, Published = DateTime.UtcNow };
-        var page1V1 = new EducationInNumbersPage { Id = Guid.NewGuid(), Slug = "page-1", Version = 1, Order = 0, Published = null };
-        var page2 = new EducationInNumbersPage { Id = Guid.NewGuid(), Slug = "page-2", Version = 0, Order = 1 };
+        var rootPage = new EducationInNumbersPage { Id = Guid.NewGuid(), Slug = null, Order = 0 };
+        var page1V0 = new EducationInNumbersPage { Id = Guid.NewGuid(), Slug = "page-1", Version = 0, Order = 1, Published = DateTime.UtcNow };
+        var page1V1 = new EducationInNumbersPage { Id = Guid.NewGuid(), Slug = "page-1", Version = 1, Order = 1, Published = null };
+        var page2 = new EducationInNumbersPage { Id = Guid.NewGuid(), Slug = "page-2", Version = 0, Order = 2 };
 
         var contentDbContextId = Guid.NewGuid().ToString();
         await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
         {
-            contentDbContext.EducationInNumbersPages.AddRange(page1V0, page1V1, page2);
+            contentDbContext.EducationInNumbersPages.AddRange(rootPage, page1V0, page1V1, page2);
             await contentDbContext.SaveChangesAsync();
         }
 
         await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
         {
             var service = CreateService(contentDbContext);
-            var newOrder = new List<Guid> { page2.Id, page1V1.Id };
+            var newOrder = new List<Guid> { rootPage.Id, page2.Id, page1V1.Id };
             var result = await service.Reorder(newOrder);
             result.AssertRight();
         }
 
         await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
         {
+            var dbRootPage = await contentDbContext.EducationInNumbersPages.FindAsync(rootPage.Id);
+            Assert.NotNull(dbRootPage);
+            Assert.Equal(0, dbRootPage.Order);
+            dbRootPage.Updated.AssertUtcNow();
+            Assert.Equal(_userId, dbRootPage.UpdatedById);
+
             // if there is an amendment, we update the original and the amendment, as the amendment could be cancelled
```

Comment:
I know this isn't part of this PR, but doesn't that mean that if the amendment is cancelled, the original's page order is still altered?

Reply:
Coooool